packages/angular-cli/models/webpack-build-node.ts

```diff
@@ -0,0 +1,86 @@
+import * as path from 'path';
+import * as webpack from 'webpack';
+
+export function getWebpackNodeConfig(projectRoot: string, environment: string, appConfig: any) {
+  const checkNodeImport = function(context: any, request: any, cb: any) {
+    if (!path.isAbsolute(request) && request.charAt(0) !== '.') {
+      cb(null, 'commonjs ' + request);
+      return;
+    }
+    cb();
+  };
+  const appRoot = path.resolve(projectRoot, appConfig.root);
+  const nodeMain = path.resolve(appRoot, appConfig.nodeMain);
+  const entryName: string = path.basename(nodeMain).replace('.ts', '');
+
+  let entry: { [key: string]: string[] } = {};
+  entry[entryName] = [nodeMain];
+
+  return {
+    devtool: 'inline-source-map',
+    resolve: {
+      extensions: ['.ts', '.js']
+    },
+    context: path.resolve(__dirname, './'),
+    output: {
+      path: path.resolve(projectRoot, appConfig.outDir),
+      filename: '[name].bundle.js',
+      libraryTarget: 'commonjs2'
+    },
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
```

Comment:
yes, it was a preloader. but the preloader config attribute is not available anymore?! I got an error. I will use `enforce: 'pre'` thank you!